frontend/src/app/main/ui/dashboard/placeholder.scss

```diff
@@ -90,3 +75,13 @@
     font-size: $fs-16;
   }
 }
+
+.placeholder-loader {
+  align-self: end;
+}
+
+.grid-empty-placeholder.loader {
+  height: 100%;
+  width: 100%;
+  justify-items: center;
+}
```

Comment:
Let's avoid this type of selector and use just a single class

Reply:
Done